mautic/mautic
# Add color and thickness options to separator in email builder

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| New feature? | Y

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This  add colour and thickness option to customise the separator in email builder.
![scrn](https://cloud.githubusercontent.com/assets/24408961/24647799/05a56130-193f-11e7-8e69-5aae3bfe5073.png)


#### Steps to test this PR:
1. Create an email and click the Builder button to open email builder
2. Add the separator and click on the separator slot
3. Check the colour and thickness option is shown in customize Slot in right side bar
